x-team/xp-ui
# Close mobile menu by clicking outside

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3533

## Description

Close mobile menu by clicking outside.

## Checklist


## Steps to Test or Reproduce

1. Go to http://localhost:9001/?path=/story/screens-and-layouts-applicantscreen-states--with-menu-links
1. Adjust viewport to mobile size and click in the hamburger icon to expand the menu
1. Now click outside of the expanded menu to see it collapsing back

## Impacted Areas in Application

- `HeaderBar`

## Screenshots

Storybook:

![2020-02-11 10 50 46](https://user-images.githubusercontent.com/131859/74243027-bb1d2200-4cbd-11ea-9b8e-bbf5d4f11629.gif)

XP Registration:

![2020-02-11 11 06 29](https://user-images.githubusercontent.com/131859/74243505-a68d5980-4cbe-11ea-9256-e091b1dcedc7.gif)
